packages/react/src/constants/CardPropTypes.js

```diff
@@ -768,7 +768,9 @@ export const CardPropTypes = {
   resizeHandles: PropTypes.array,
   /** Optional callback function that is passed an onChange function and the original cardConfig object.
    * This allows additional information to be passed to be used in the Card Editor for this type.
-   * You need to return an array of child objects with a header: {title, tooltip: {tooltipText: PropTypes.string}} and content element to render * */
+   * You need to return an array of child objects with a header: {title, tooltip: {tooltipText: PropTypes.string}} and content element to render
+   * DEPRECATED: we recommend doing this dynamically with the CardEditor onRenderCardEditForm property
```

Comment:
```suggestion
   * we recommend doing this dynamically with the CardEditor onRenderCardEditForm property
```

Let's remove DEPRECATED since this method can still be used within the new `onRenderCardEditForm`